src/sentry/api/endpoints/chunk_upload.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+# from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry import options
+from sentry.models import FileBlob
+from sentry.models.file import DEFAULT_BLOB_SIZE
+from sentry.api.base import Endpoint
+from sentry.api.bases.project import ProjectReleasePermission
+
+UPLOAD_ENDPOINT_SETTING = options.get('system.upload-url-prefix')
```

Comment:
Don't do this at module load time. Query these inside of `ChunkUploadEndpoint.get`.